include/deal.II/lac/relaxation_block.h

```diff
@@ -350,11 +334,6 @@ class RelaxationBlockJacobi : public virtual Subscriptor,
    * the dst() vector to zero before calling the Tstep() method.
    */
   void Tvmult (VectorType &dst, const VectorType &rhs) const;
-
-  /**
-   * Return the memory allocated in this object.
-   */
-  std::size_t memory_consumption() const;
```

Comment:
what do we do with memory_consumption? I think it would be useful to have.

Reply:
Since this has never been used I would prefer to just remove it. We can implement it later if the need arises.